UML-Diagram/ExceptionErrorHandling.puml

```diff
@@ -0,0 +1,44 @@
+@startuml
+
+package "Exception Handling" {
+    enum Severity {
+        {static} + LOW
+        {static} + MEDIUM
+        {static} + HIGH
+        {static} + CRITICAL
+    }
+
+    abstract class GlobalException {
+        + Message : String
+        + Severity : Severity
+        + GlobalException(String message, Severity severity)
+    }
+
+    class BookNotFoundException {
+        + BookNotFoundException(String message)
+    }
+
+    class PatronNotFoundException {
+        + PatronNotFoundException(String message)
+    }
+
+    Exception <|-- GlobalException
+    GlobalException <|-- BookNotFoundException
+    GlobalException <|-- PatronNotFoundException
+    GlobalException --> Severity
+}
+
+package "Error Handling" {
```

Comment:
Packages could be in different .puml archives, you can reference Severity enum from the original archive by using:

!ExceptionErrorHandling.puml

int he ErrorHangling.puml archive so can communicate different packages on a same place, an example for this its on: [Master-UML.puml](https://github.com/programacion-6/Opcion1CanelitaTeam/blob/docs/UML-Diagram/Master-UML.puml)

Reply:
Yeah! I didn't catch you before. But now it's like this in the` Master-UML.puml`
![image](https://github.com/user-attachments/assets/a9425069-8938-4bbe-a5f4-aa56f1b4c844)

Reply:
Yes, that was what I was referring to.